Runtimes/Core/cmake/modules/CompilerChecks/ptr_size

```diff
@@ -0,0 +1 @@
+__SIZEOF_POINTER__
```

Comment:
Should this be gitignored? Looks like `PlatformInfo.cmake` writes this when it's needed. Or maybe `PlatformInfo.cmake` doesn't need to write it.

Reply:
Right, but it's also created there:

```
  file(WRITE ${CMAKE_BINARY_DIR}/cmake/CompilerChecks/ptr_size "__SIZEOF_POINTER__")
  set(size_check_command ${CMAKE_C_COMPILER} -x c -P -E -o - ${CMAKE_BINARY_DIR}/cmake/CompilerChecks/ptr_size)
```

I'm thinking it should be committed, or written when needed, but probably not both.

Reply:
Yeah, this file isn't necessary. Since we have C enabled, I can probably just use `CMAKE_SIZEOF_VOID_P` and skip this. I was having some issues with it before where it was empty/wrong with older versions of CMake, but it looks like it's behaving correctly now.